controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/application/v4/CostResource.java

```diff
@@ -0,0 +1,27 @@
+package com.yahoo.vespa.hosted.controller.api.application.v4;
+
+import com.yahoo.vespa.hosted.controller.api.application.v4.model.CostMonths;
+import com.yahoo.vespa.hosted.controller.api.application.v4.model.CostResult;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
+@Path("")  //Ensures that the produces annotation is inherited
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public interface CostResource {
```

Comment:
Author should be added, and javadoc. Also goes for the other classes.

Reply:
Added the author tag, but skipped javadoc. 0 classes in that module have it, and they are just use to parse JSON data.